oap-server/server-starter/src/main/resources/ui-initialized-templates/pulsar/pulsar-cluster.json

```diff
@@ -0,0 +1,447 @@
+                    "showYAxis": true
+                  },
+                  "widget": {
+                    "name": "storage_read_rate",
+                    "title": "Storage Read Rate"
+                  }
+                },
+                {
+                  "x": 12,
+                  "y": 30,
+                  "w": 12,
+                  "h": 10,
+                  "i": "16",
+                  "type": "Widget",
+                  "id": "16-0-16",
+                  "metricTypes": [],
+                  "metrics": [],
+                  "metricMode": "Expression",
+                  "moved": false,
+                  "expressions": [
+                    "meter_pulsar_storage_logical_size.sum(['cluster','node'])/1024/1024"
+                  ],
+                  "typesOfMQE": [
+                    "TIME_SERIES_VALUES"
+                  ],
+                  "graph": {
+                    "type": "Line",
+                    "step": false,
+                    "smooth": false,
+                    "showSymbol": true,
+                    "showXAxis": true,
+                    "showYAxis": true
+                  },
+                  "widget": {
+                    "name": "storage_logical_size",
+                    "title": "Storage Logical Size"
+                  }
+                },
+                {
+                  "x": 0,
+                  "y": 40,
+                  "w": 12,
+                  "h": 10,
+                  "i": "17",
+                  "type": "Widget",
+                  "id": "16-0-17",
+                  "metricTypes": [],
+                  "metrics": [],
+                  "metricMode": "Expression",
+                  "moved": false,
+                  "expressions": [
+                    "meter_pulsar_storage_write_rate.sum(['cluster','node'])"
+                  ],
+                  "typesOfMQE": [
+                    "TIME_SERIES_VALUES"
+                  ],
+                  "graph": {
+                    "type": "Line",
+                    "step": false,
+                    "smooth": false,
+                    "showSymbol": true,
+                    "showXAxis": true,
+                    "showYAxis": true
+                  },
+                  "widget": {
+                    "name": "storage_write_rate",
+                    "title": "Storage Write Rate"
+                  }
+                },
+                {
+                  "x": 0,
+                  "y": 30,
+                  "w": 12,
+                  "h": 10,
+                  "i": "18",
+                  "type": "Widget",
+                  "id": "16-0-18",
+                  "metricTypes": [
+                    "readLabeledMetricsValues"
+                  ],
+                  "metrics": [
+                    "meter_pulsar_storage_size/1024/1024"
+                  ],
+                  "metricMode": "General",
+                  "moved": false,
+                  "expressions": [],
```

Comment:
![image](https://github.com/apache/skywalking/assets/10701915/177e9822-39da-4699-86c7-d170ee1047d1)
`expression` is empty, causing the `storage size` graph to load exceptionally.